pkg/internal/cryptosetup/cryptosetup_test.go

```diff
@@ -187,3 +187,16 @@ func (m mockKeyManager) Get(keyID string) (interface{}, error) {
 func (m mockKeyManager) Rotate(kt kmsservice.KeyType, keyID string) (string, interface{}, error) {
 	panic("implement me")
 }
+
+func (m mockKeyManager) ExportPubKeyBytes(keyID string) ([]byte, error) {
+	return nil, nil
+}
+
+func (m mockKeyManager) PubKeyBytesToHandle(pubKey []byte, keyType kmsservice.KeyType) (interface{}, error) {
+	return nil, nil
+}
+
+func (m mockKeyManager) ImportPrivateKey(
+	privKey interface{}, kt kmsservice.KeyType, opts ...kmsservice.PrivateKeyOpts) (string, interface{}, error) {
+	return "", nil, nil
+}
```

Comment:
Is this still needed now that you're using the other mockkms?

Reply:
I'm using this in a few places, to replicate the same error that your tests were checking for.